website/docs/tutorials/guides/testing.md

```diff
@@ -443,18 +443,68 @@ Deploys a contract given a path relative to a Protostar project root. The sectio
 ### `start_prank`
 
 ```python
-def start_prank(caller_address: int) -> None: ...
+def start_prank(caller_address: int, target_contract_addr: Optional[int] = None) -> Callable: ...
 ```
 
-Changes caller address until [`stop_prank`](#stop_prank) cheatcode is used.
+Changes caller address until the returned callable is called. If `target_contract_addr` specified, prank affects only the contract with the specified address. Otherwise prank affects the current contract.
 
-### `stop_prank`
+#### With target
+```cairo title="./pranked_contract.cairo"
+%lang starknet
 
-```python
-def stop_prank() -> None: ...
+from starkware.starknet.common.syscalls import (get_caller_address)
+
+@view
+func assert_pranked{syscall_ptr : felt*}():
+    let (caller_addr) = get_caller_address()
+    with_attr error_message("Not pranked!"):
+        assert caller_addr = 123
+    end
+    return ()
+end
 ```
+```cairo title="Remote assert passes"
+@contract_interface
+namespace Pranked:
+    func assert_pranked() -> ():
+    end
+end
+
+@external
+func test_remote_prank{syscall_ptr : felt*, range_check_ptr}():
+    alloc_locals
+    local contract_address : felt
+    %{ 
+        ids.contract_address = deploy_contract("./pranked_contract.cairo").contract_address 
+        stop_prank_callable = start_prank(123, target_contract_addr=ids.contract_address)
+    %}
+    # Does not raise error
+    Pranked.assert_pranked(contract_address=contract_address)
+    %{ stop_prank_callable() %}
+    return ()
+end
+``` 
 
-Resets caller address. Always used with [`start_prank`](#start_prank).
+#### Locally
+```cairo title="Local assert passes"
```

Comment:
Local assert should be the first example. Unit tests are more common than integration tests.